### PR TITLE
Add Roborock S8 Pro Ultra

### LINKED ISSUE #1864: Unsupported device Roborock S8 Pro Ultra
**Device information:**

  - Name(s) of the device: Roborock S8 Pro Ultra
  - Link: https://de.roborock.com/pages/roborock-s8-pro-ultra

Use `miiocli device --ip <ip address> --token <token> info`.

Model: roborock.vacuum.a70
Hardware version: Linux
Firmware version: 4.3.5_1786

### PR DESCRIPTION
**Pull request by spangenberg**
Fixes #1864

### CODE REVIEW

**Comment by spangenberg** [comment]
`CI / Perform linting checks` are unrelatedly failing and are also currently happening on master branch.

```@spangenberg ➜ /workspaces/python-miio (master) $ poetry run pre-commit run flake8 --all-files
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

miio/devtools/pcapparser.py:54:13: B036 Don't except `BaseException` unless you plan to re-raise it.
miio/integrations/lumi/gateway/alarm.py:97:13: B038 editing a loop's mutable iterable often leads to unexpected results/bugs
miio/integrations/lumi/gateway/devices/subdevice.py:334:13: B038 editing a loop's mutable iterable often leads to unexpected results/bugs```

File: miio/integrations/roborock/vacuum/vacuum.py [hunk 1]
```diff
@@ -74,6 +74,7 @@
 ROCKROBO_T7SPLUS = "roborock.vacuum.a23"
 ROCKROBO_S7_MAXV = "roborock.vacuum.a27"
 ROCKROBO_S7_PRO_ULTRA = "roborock.vacuum.a62"
+ROCKROBO_S8_PRO_ULTRA = "roborock.vacuum.a70"
```
Comment (herrniffler) [inline miio/integrations/roborock/vacuum/vacuum.py:77]: you can also add it to  `AUTO_EMPTY_MODELS` see line 120

Comment (spangenberg) [reply miio/integrations/roborock/vacuum/vacuum.py:77]: Absolutely, thanks for pointing that out!

**Review by rytilahti — APPROVED** [review]
Thanks :+1:

**Pull request merged**